### ISSUE TITLE
Enable development on Windows

### ISSUE THREAD

**Issue by leenattress** [anishkny/realworld-dynamodb-lambda#67]
```
> realworld-dynamodb-lambda@1.0.0 postinstall C:\Users\deept\Documents\Node\realworld-dynamodb-lambda
> serverless dynamodb install 2>&1 | awk 'NF'

awk: 'NF'
awk: ^ invalid char ''' in expression
```

Is there any way to make this work on windows? I suspect its the single speech marks.

**Comment by gabrielcrowe** [comment]
oops, i was right.

`"postinstall": "serverless dynamodb install 2>&1 | awk \"NF\""`

**Comment by gabrielcrowe** [comment]
I have no idea how to contribute this fix myself, or i would. :/

**Comment by gabrielcrowe** [comment]
Well now, i had no idea this was so geared up for linux development.

looks like the startup scripts are .sh scripts, i guess as a windows person i cant use this. :/

**Comment by anishkny** [comment]
@gabrielcrowe agreed. This repo was setup more with POSIX in mind. I started a branch to experiment with Windows Node.js development: [`azure-pipelines`](https://github.com/anishkny/realworld-dynamodb-lambda/tree/azure-pipelines), [build](https://dev.azure.com/anishkarandikar/realworld-dynamodb-lambda/_build/results?buildId=131)

**Comment by f0rfun** [comment]
Yeah, getting stuck until I saw these issues.. argh.

**Comment by leenattress** [comment]
for your info, i remember getting around this using WSL

https://docs.microsoft.com/en-us/windows/wsl/install-win10

**Comment by leenattress** [comment]
And git bash, part of windows git package.
